gatsby-node.esm.js

```diff
@@ -7,7 +7,6 @@ import path from "path";
 import webpack from "webpack";
 import SchemaCustomization from "./schema";
 import { getApiContestData } from "./netlify/util/getContestsData";
-const { token } = require("./netlify/_config");
```

Comment:
it might be nice to fall back to the original token if the new one isn't set (thinking mostly of devs having to update all their own `.env` files now too) but not a blocker for me

Reply:
Fair point